src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/GoogleCalendar/_Layout.cshtml

```diff
@@ -0,0 +1,9 @@
+@{
```

Comment:
why do we need this?

Reply:
I've decoupled the google integration from the Manage controller, so I want the views in GoogleCalendar to use the layout from Manage views and not from Shared

Reply:
A better option, in this case, would be to add _ViewStart to the GoogleCalendar Views folder

Reply:
But if we want to have only one view for the user, we could get rid of the entire Google Calendar controller and just implement that logic in the Manage controller. One more dependency on service there won't be such a big deal.

Reply:
@Mejtika @Kamil4921 - do we have any update here?

Reply:
@WTobor We're trying